chaomoua91/onPointFlix
# Msg 74 home page/sidebar

### ✨ Why this feature?
- Explain why this feature/fix is needed
(example: We need to add a sidebar to contain a user's movie watch list)

### 🛠 Changes being made
Here give examples of the changes you've made in this pull request. Include an itemized list if you can.
example:
- Added axios to fetch tmdb api data
- Added shadcn UI library to style components

### 🧪 Testing
Write a list of quality assurance testing instructions for your reviewer to double check your work locally
1. Git fetch and checkout my branch
2. Run `npm install` then `npm run dev`
3. Navigate to the home page and verify that the movies appear
